### PR TITLE
Enhancement/kmnr

### LINKED ISSUE #13: Play Music in Voice Channel?
Purely for fun, could add songs to queue, pause, skip songs

### PR DESCRIPTION
**Pull request by GRNCheetah**
Closes #13

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,11 +11,14 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "dependencies": {
+    "@discordjs/opus": "^0.1.0",
     "@types/shortid": "^0.0.29",
     "discord-command-parser": "^1.3.3",
     "discord.js": "^12.1.1",
+    "ffmpeg-static": "^4.1.0",
```
Comment (ClayMav) [inline package.json:18]: ?

File: src/commands/kmnr.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    voiceConn.on("reconnecting", () => {
+      logBot.warn("Attempting to reconnect to voice channel");
+    });
+
+    // Start playing them sick beats
+    // When you create a broadcast, it is added to an array here
+    // client.voice.createBroadcast()
+    // We add it to a var so we can manipulate it
+    const kmnrBroadcast = client.voice.createBroadcast();
```
Comment (ClayMav) [inline src/commands/kmnr.ts:30]: move broadcast out of here and allow for subscribing

**Review by ClayMav — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (GRNCheetah) [reply package.json]: Removed

**Review by ClayMav — APPROVED** [review]
LGTM 👌

**Pull request merged**